modules/currency-l1/src/main/scala/org/tessellation/currency/l1/modules/HttpApi.scala

```diff
@@ -110,7 +109,7 @@ sealed abstract class HttpApi[
           nodeRoutes.publicRoutes <+>
           metricRoutes <+>
           targetRoutes <+>
-          routes.as(HttpRoutes.empty[F]).getOrElse(currencyRoutes.publicRoutes)
+          routes.map(_.publicRoutes).getOrElse(currencyRoutes.publicRoutes)
```

Comment:
So either data routes or currency routes. Is that correct?

Reply:
yes, that's intended. I have confirmed that with Alex and Ryle yesterday.